catalyst-toolbox-cli/src/cli/notifications/api_params.rs

```diff
@@ -0,0 +1,11 @@
+use reqwest::Url;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct ApiParams {
+    #[structopt(long, default_value = "https://cp.pushwoosh.com/json/1.3/")]
+    pub api_url: Url,
+    #[structopt(long)]
+    pub access_token: String,
```

Comment:
I am not a big fan of passing a token as a command line argument... IMO it is better to read it from stdin.

Reply:
We already read the content path from there, but probably is better to switch them. Will do.

Reply:
Reading content from the cli as arg could be pretty cumbersome. I also added a feature so content can be either a plain string or a multilanguage structure as defined by pushwoosh API which would make it even more weird to use.
Now there is always the possibility to use the json file with all the configuration instead of the args command, that would solve this "issue".